### DISCUSSION TITLE
Is there an "current or expected functionality" list somewhere?

### DISCUSSION THREAD

**Discussion by StingyJack** [CoreWCF/CoreWCF#281] [Q&A] [answered]
I saw the feature roadmap sticky but cant find anything that says what the key functionalities are that are expected to be available in the first or subsequent releases. Or where any gaps are that need bridging. Am I missing something obvious?

**Comment by birojnayak** [comment, marked as answer]
@StingyJack the Roadmap is a bigger umbrella to cover all features and getting community votes.. with every GA (https://github.com/CoreWCF/CoreWCF/milestones) we are planning to release feature availability and some writeup for community to understand. If you want some feature to be available or prioritized please vote on the Roadmap, if its not available please comment on the roadmap issue or create an issue, we will put them into appropriate bucket or address them in monthly community meeting (https://github.com/CoreWCF/CoreWCF/wiki/Community-Sync-up-meetings) . @mconnew any other thought ?
